cmd/end2endtest/helpers.go

```diff
@@ -602,15 +601,7 @@ func faucetPackage(faucet, faucetAuthToken, myAddress string) (*models.FaucetPac
 }
 
 func matchResults(results, expectedResults [][]*types.BigInt) bool {
-	// iterate over each question to check if the results match with the expected results
-	for i, result := range results {
-		for q, r := range result {
-			if !(expectedResults[i][q].String() == r.String()) {
-				return false
-			}
-		}
-	}
-	return true
+	return reflect.DeepEqual(results, expectedResults)
```

Comment:
Just to confirm, using reflect is not slower?

Reply:
i don't know. but in any case it's executed just once per test :shrug: